common/djangoapps/student/cookies.py

```diff
@@ -107,7 +112,7 @@ def set_logged_in_cookies(request, response, user):
     user_info_cookie_is_secure = request.is_secure()
 
     response.set_cookie(
-        settings.EDXMKTG_USER_INFO_COOKIE_NAME,
+        settings.EDXMKTG_USER_INFO_COOKIE_NAME.encode('utf-8'),
```

Comment:
Why is it OK to encode the unicode strings as utf-8? Couldn't that cause a loss of information?